run_tests.py

```
@@ -0,0 +1,31 @@
+Test Runner Script for PrismOS Installation Tests
+"""
+import subprocess
+import sys
+
+def run_tests():
+    """Run all tests and display summary"""
+    print("🚀 Running PrismOS Installation Tests...")
+    print("=" * 50)
+    
+    # Run unit tests
+    result = subprocess.run([
```

Ruff (0.14.0)

`subprocess` call: check for execution of untrusted input

(S603)